src/glview/GLView.cc

```diff
@@ -382,6 +394,49 @@ void GLView::showCrosshairs(const Color4f& col)
   glEnd();
 }
 
+void GLView::showObject(const SelectedObject &obj, const Vector3d &eyedir)
+{
+  auto vd = cam.zoomValue()/200.0;
```

Comment:
I assume this should keep the markers relative to the model in size regardless of zoom level? For me it starts making the markers quite large when zooming out. Maybe also something to do with orthogonal/perspective? I don't think it's a showstopper, the sizes worked ok with the tests so far.

Reply:
no subjective issue here. could this relate to QGLView.cc:420

const auto height = dist * tan_degrees(cam.fov ...